include/zephyr/net/hostname.h

```diff
@@ -108,8 +109,31 @@ int net_hostname_set_postfix(const uint8_t *hostname_postfix,
 static inline int net_hostname_set_postfix(const uint8_t *hostname_postfix,
 					   int postfix_len)
 {
-	ARG_UNUSED(hostname_postfix);
```

Comment:
Please don't remove `ARG_UNUSED()` macros here. And add them to the new function too.

Reply:
Fixed.